build/index.js

```diff
@@ -95,9 +100,64 @@ const createIconDocs = async (iconFile) => {
 
 const createPackageDocs = (packageExports) => {
   return `
-{-| This package is a port of Steve Schoger's Zondicons collection
+{-| A port of Steve Schoger's Zondicons collection
 
-# Icons
+## Customizing Icons Color
+For convenience sake by default the icons will take the current color of their parent element.
+If this is not the desired behaviour it can be changed using the CSS \`fill\` propery and setting it to the desired value by either using a class or an inline style.
```

Comment:
I would add some things
> For convenience sake by default the icons will take the current text color of their parent element. So the preferred way to set colors would be to set something like `color: green` in CSS.
If this is not the desired behaviour, a different fill mode can be set via the fill attribute, or CSS fill property.